tracer/build/PackageVersionEntrySchema.json

```diff
@@ -64,15 +64,19 @@
                       "uniqueItems": true
                     }
                   ]
+                },
+                "SkipAlpine": {
+                  "type": "boolean",
+                  "description": "If true, packages in the range will not built on Alpine"
```

Comment:
nit: will not be built on Alpine

Reply:
Thanks will fix in a separate PR to avoid another CI cycle 🙂